src/coalesce-vue-vuetify3/src/components/display/c-table.vue

```diff
@@ -92,10 +105,51 @@ export default defineComponent({
     props: { required: false, type: Array as PropType<Array<string>> },
     admin: { required: false, type: Boolean },
     editable: { required: false, type: Boolean },
-    extraHeaders: { required: false, type: Array as PropType<Array<string>> },
+    extraHeaders: {
+      required: false,
+      type: Array as
+        | PropType<Array<{ header: string; isFixed: boolean }>>
+        | PropType<Array<string>>,
+    },
     loaders: { required: false, type: Object },
   },
 
+  setup() {
```

Comment:
Unsure what your testing policy is & if there should be tests for this?

Reply:
This is fine without tests. Would be fairly hard to write automated tests, and very easy to manually test.